scripts/build-util.sh

```diff
@@ -81,6 +86,9 @@ module_build() ( # <submodule> [configure-arg..]
         "${MAKE}"
         echo "==>  Installing ${name}"
         "${MAKE}" install
+        if [ "$clean_after_install" = "true" ] ; then
```

Comment:
Based on the original patch, I assumed the intent was to clean up after all the submodules rather than just riscv-gnu-toolchain:

```suggestion
        if [ -n "$CLEANAFTERINSTALL" ] ; then
```

which makes parsing `--clean-after-install` unnecessary.

If a particular submodule needs to be excluded, the variable could be overridden at the call site:
```sh
CLEANAFTERINSTALL= module_build ...
```